CHANGELOG.md

```diff
@@ -17,6 +17,14 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.13.3] - 2023-08-14
+### Fixed
+- Fixed bug in `ViewApply.properties` had type hint `ConnectionDefinition` instead of `ConnectionDefinitionApply`.
```

Comment:
What was the bug? I (personally) don't think we need to update the changelog with updates/fixes to type annotations 😄

Reply:
I think it is still nice to include, as it makes it clear for posterity that this was a mistake.

Reply:
...then the add `pydoclint` PR will be 1000+ lines in the changelog 😂